src/components/ManageUsers/manage-users.css

```
@@ -0,0 +1,84 @@
+.usersearch__h1 {
+  font-size: 16px;
+  color: #e3893e;
```

```suggestion
  color: var(--color-primary);
```

Change everywhere where you are using our orange color

---

Has done for manage-users.css